### PR TITLE
add: druidfetcher support for different configId

### PR DESCRIPTION
**Pull request by s0nicboOm**
Update config to support multiple DruidFetcher

### CODE REVIEW

File: numalogic/tools/types.py [hunk 1]
```diff
@@ -17,11 +17,11 @@
 from numalogic.base import TorchModel, BaseThresholdModel, BaseTransformer
 
 try:
-    from redis.client import AbstractRedis
+    from redis.client import Redis
```
Comment (ab93) [inline numalogic/tools/types.py:20]: Any specific reason?

**Review by ab93 — APPROVED** [review]
Minor comment

File: numalogic/tools/types.py (hunk 1 above)
Comment (s0nicboOm) [reply numalogic/tools/types.py:20]: Yes. I was following what are the different clients that come from redis-py.  They expose mainly two clients:
Redis and StrictRedis
links: 
1) https://stackoverflow.com/questions/19021765/redis-py-whats-the-difference-between-strictredis-and-redis
2) https://devpress.csdn.net/python/62fd2d2ac677032930802e87.html 
3) https://docs.ros.org/en/hydro/api/redis/html/redis.client-module.html

Comment (s0nicboOm) [reply numalogic/tools/types.py:20]: This is why I moved to Redis. Eventually we should move to StrictRedis

**Pull request merged**